products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp

```diff
@@ -192,20 +198,26 @@ namespace app::focus
     {
         switch (currentTimerPhase) {
         case FocusTimerPhase::FocusTimeEnded:
-        case FocusTimerPhase::ShortBreakTimeEnded:
+        case FocusTimerPhase::BreakTimeEnded:
             return true;
         default:
             return false;
         }
     }
 
+    bool FocusTimerPresenter::isTimeForLongBreak()
+    {
+        return (longBreakDuration.count() > 0) &&
+               (((allFocusSessionsCount - focusSessionsLeft) % longBreakOccurrence) == 0);
```

Comment:
I'd factor out `allFocusSessionsCount - focusSessionsLeft` to some separate variable (`focusSessionsElapsed`?), that way it will be easier to read

Reply:
done